packages/client/src/generators.test.ts

```diff
@@ -0,0 +1,12 @@
+import { generateCodeVerifier, generateCodeChallenge } from './generators';
+
+test('generate codeVerifier', () => {
+  const verifier = generateCodeVerifier();
+  expect(verifier.length).toEqual(96);
```

Comment:
nit: it would be better to give all the constant number a readable const definition. 
i.e. 
```js
const CODE_VERIFIER_LEN = 96;
const HASH_ENCODED_LEN = 43
```